### PR TITLE
write document api update profile

### PR DESCRIPTION
**Pull request by ghost**
## Description
write api update profile
## Reference Requirements
- Task ID: TI-1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-sonle** [comment]
LGTM

File: docs/api-references/user.md [hunk 1]
```diff
@@ -180,4 +180,71 @@ Register user
...
+
+#### Response - Success
+``` json
+{
+    "meta": {
+        "status": "Successfully",
+        "code": 200
+    },
+    "data": {
+        "data": {
```
Comment (at-tuanphan) [inline docs/api-references/user.md]: 2 cấp data rồi em

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**